code/__DEFINES/misc.dm

```diff
@@ -155,11 +155,36 @@
 #define ASSEMBLY_UNLOCKED 1
 #define ASSEMBLY_LOCKED 2
 
+// RESEARCH UPGRADES DEFINES //
+
 // Matrix CAS Upgrades
 #define MATRIX_DEFAULT 0
 #define MATRIX_NVG 1
 #define MATRIX_WIDE 2
 
+#define RESEARCH_UPGRADE_EXCLUDE_BUY 0
+#define RESEARCH_UPGRADE_CATEGORY 0.5 //lord forgive me
```

Comment:
I think if you're gonna do the silly thing and mix tiers and modes, you can at least use -1, -2, etc

Reply:
didnt want to clutter things with vars that get used once in a morbillion. Thanks